docs/api/ethereum.md

```diff
@@ -1352,7 +1352,7 @@ A class instance `variable` holding the address of ledger's smart contract on th
 An `asynchronous` class instance `function` which grants management permissions for this ledger to a third party `accountId`.
 
 ::: warning
-The `MANAGE_ABILITIES` ledger ability is required to perform this function.
+The `MANAGE_ABILITIES` ledger super ability is required to perform this function.
```

Comment:
change to `super ledger ability`

Reply:
This can sound like "super ledger" ability instead of super "ledger ability". While if we say ledger super ability there is no confusion? @typwrtr What do you think?